src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/navbar-logo.html

```diff
@@ -11,7 +11,7 @@
 <a class="navbar-brand logo" href="{{ href }}">
   {# get all the brand information from html_theme_option #}
   {% set is_logo = "light" in theme_logo["image_relative"] %}
-  {% set alt = theme_logo.get("alt_text", "Logo image") %}
+  {% set alt = theme_logo.get("alt_text", docstitle + " - Home" if not theme_logo.get("text") else "") %}
```

Comment:
I think this makes everything more consistent

Reply:
I like this logic. Suggestion is to add an expanatory comment and also reverse the order of the logic (seems easier to grok to me)

```suggestion
  {# use alt_text if given. If not, only add alttext if no add'l branding text given. #}
  {% set alt = theme_logo.get("alt_text", "" if theme_logo.get("text") else f"{docstitle} - Home") %}
```

Reply:
Hmm when I used an f-string the first time around, the build script failed. I'll commit this suggestion and see if it builds.

Reply:
ah, oops, forgot this was within jinja.  yeah need to switch back to using `+`